docs/products/m3db/howto/write-go.rst

```diff
@@ -0,0 +1,42 @@
+Write Data to M3DB with Go
+--------------------------
+
+This example writes some data to an M3DB service from Go, making use of the Prometheus write features.
+
+Variables
+'''''''''
+
+These are the placeholders you will need to replace in the code sample:
+
+==================      ==========================================================
```

Comment:
Should we always use the `list-table` structure?

Reply:
I'm not sure it matters (this might bite us later though, you're quite right!)